src/platform/test.rs

```diff
@@ -198,15 +193,9 @@ fn with_n_fds(n: usize, size: usize) {
     let (super_tx, super_rx) = platform::channel().unwrap();
 
     let data: Vec<u8> = (0..size).map(|i| (i % 251) as u8).collect();
-    let thread = {
-        let data = data.clone();
-        thread::spawn(move || {
-            super_tx.send(&data[..], sender_fds, vec![]).unwrap();
-        })
-    };
+    super_tx.send(&data[..], sender_fds, vec![]).unwrap();
     let (received_data, received_channels, received_shared_memory_regions) =
         super_rx.recv().unwrap();
-    thread.join().unwrap();
```

Comment:
I think this change needs to be reverted.

Reply:
Nevermind; it seems to have originated from https://github.com/servo/ipc-channel/commit/b473acbf09b09c86485daf0ce7294f7d487984b6 and seems independent of the windows changes in this PR.